get_member_status_by_email_test.py

```diff
@@ -0,0 +1,29 @@
+from get_member_status_by_email import get_member_status
+
+
+def display_members(members):
+    if (len(members)) == 0:
+        print("member does not exist")
+
+    for member in members:
+        print("==============")
+        print("member for email : ", member.email)
+
+        if member.member_status is not None:
+            print("member exists")
+            print("status : ", member.member_status)
+            print("type : ", member.member_type)
+            print("password last change date : ",
+                  member.password_last_change_date)
+        else:
+            print("member does not exist")
+
+
+found_members = get_member_status('aaa@email.com')
+display_members(found_members)
+
+found_members = get_member_status('mmm@email.com')
+display_members(found_members)
+
+found_members = get_member_status('aaas49807@email.com')
+display_members(found_members)
```

Comment:
Place test files in a tests directory

Reply:
i would like to do this. 
once i did it, the import did not work. the test imports the production code.
in order to test i run the test file. if code is in higher level of directories structure - test doesn't import the file.
if you have a solution for this problem. i will be happy to implement it.
thanks :)

Reply:
@yishaiz use the following method to add folders to the python modules search path - sys.path.append